osidb/mixins.py

```diff
@@ -148,68 +149,75 @@ class Meta:
         abstract = True
 
 
+class Alert(ValidateMixin):
```

Comment:
It was added mostly to not allow incorrect alert types, as there was a test case for that (`TestModelDefinitions::test_alert_incorrect_type`). But it's true that this case would be a programmer error since alerts are not created by users. So I will remove this test case as well.